src/test/java/io/crate/jmx/integrationtests/MetricsITest.java

```diff
@@ -44,17 +44,38 @@ public static void fetchMetrics() throws Exception {
 
     @Test
     public void testQueryStatsMetrics() {
-        assertThat(METRICS_RESPONSE, containsString("crate_query_duration_seconds{query=\"Select\",} 0.0\n"));
+        // crate_query_duration_seconds
+        String metricStr = "crate_query_duration_seconds{query=\"Select\",} ";
+        assertThat(METRICS_RESPONSE, containsString(metricStr));
+        assertNonZeroQueryMetric(metricStr);
+
         assertThat(METRICS_RESPONSE, containsString("crate_query_duration_seconds{query=\"Update\",} 0.0\n"));
         assertThat(METRICS_RESPONSE, containsString("crate_query_duration_seconds{query=\"Delete\",} 0.0\n"));
         assertThat(METRICS_RESPONSE, containsString("crate_query_duration_seconds{query=\"Insert\",} 0.0\n"));
-        assertThat(METRICS_RESPONSE, containsString("crate_query_duration_seconds{query=\"Overall\",} 0.0\n"));
 
-        assertThat(METRICS_RESPONSE, containsString("crate_queries{query=\"Select\",} 0.0\n"));
+        metricStr = "crate_query_duration_seconds{query=\"Overall\",} ";
+        assertThat(METRICS_RESPONSE, containsString(metricStr));
+        assertNonZeroQueryMetric(metricStr);
+
+        // crate_queries
+        metricStr = "crate_queries{query=\"Select\",} ";
+        assertThat(METRICS_RESPONSE, containsString(metricStr));
+        assertNonZeroQueryMetric(metricStr);
+
         assertThat(METRICS_RESPONSE, containsString("crate_queries{query=\"Update\",} 0.0\n"));
         assertThat(METRICS_RESPONSE, containsString("crate_queries{query=\"Delete\",} 0.0\n"));
         assertThat(METRICS_RESPONSE, containsString("crate_queries{query=\"Insert\",} 0.0\n"));
-        assertThat(METRICS_RESPONSE, containsString("crate_queries{query=\"Overall\",}"));
+
+        metricStr = "crate_queries{query=\"Overall\",} ";
+        assertThat(METRICS_RESPONSE, containsString(metricStr));
+        assertNonZeroQueryMetric(metricStr);
+    }
+
+    private void assertNonZeroQueryMetric(String metricStringPattern) {
```

Comment:
a bit missleading name as the pattern also matches with zero (0.0). maybe just leaving out the numbers from the checked string is better?